planning/mapless_architecture/autoware_mission_lane_converter/include/autoware/mission_lane_converter/mission_lane_converter_node.hpp

```diff
@@ -0,0 +1,196 @@
+// Copyright 2024 driveblocks GmbH
+// driveblocks proprietary license
+#ifndef AUTOWARE__MISSION_LANE_CONVERTER__MISSION_LANE_CONVERTER_NODE_HPP_
+#define AUTOWARE__MISSION_LANE_CONVERTER__MISSION_LANE_CONVERTER_NODE_HPP_
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "autoware_auto_planning_msgs/msg/path.hpp"
+#include "autoware_auto_planning_msgs/msg/trajectory.hpp"
```

Comment:
Messages should be from `autoware_planning_msgs`.

Reply:
Done!

Reply:
I guess the commit was not pushed :sweat_smile:

Reply:
Yes, I will push all the changes when the internal review is finished!